app/views/shopify_app/sessions/enable_cookies.html.erb

```diff
@@ -24,7 +24,7 @@
       id: 'redirection-target',
       data: {
         target: {
-          myshopifyUrl: "https://#{current_shopify_domain}",
+          myshopifyUrl: "#{ShopifyAPI::Context.host_scheme}://#{current_shopify_domain},
```

Comment:
I think when we're pointing to `current_shopify_domain`, we should always be using https - what is http in this scenario is the app only, right?

Reply:
that's right! Great call out